### DISCUSSION TITLE
Format current values after MQTT Update

### DISCUSSION THREAD

**Discussion by hoetzgit** [bellrichm/weewx-jas#40] [Q&A] [answered]
Hello, I have a question about the daily overview. When the page is generated, all current values are formatted correctly. However, after an update is done via MQTT, then the raw data is loaded from the broker and is no longer formatted correctly. I think I can parameterize this for sure. Can someone tell me where I can make the correct settings?

Thanks a lot
Henry

screenshot:
![image](https://user-images.githubusercontent.com/41108116/188073181-642c6132-890b-4556-9317-6f5ab838c8fc.png)

**Comment by bellrichm** [comment, marked as answer]
Unfortunately you can’t do it yet. It’s high on my list #42

**Reply by hoetzgit** [reply]
Thank you for your answer. Then I look forward to the implementation.
